backend/service/aws/aws_test.go

```diff
@@ -343,6 +345,21 @@ func TestDescribeAutoscalingGroupsErrorHandling(t *testing.T) {
 	assert.Error(t, err2)
 }
 
+func TestErrorIntercept(t *testing.T) {
+	c := &client{}
+	{
```

Comment:
do the test scenarios ned to be wrapped in `{`, `}`

Reply:
it scopes the vars so you don't accidentally using a value from a previous scenario. i like to use it for really simple tests instead of splitting them out into individual tests and repeating the setup.

Reply:
oh nice! good to know